__tests__/app.test.js

```diff
@@ -222,4 +222,103 @@ describe('app', () => {
                 })
         })
     })
+    describe('PATCH /api/articles/:article_id', () => {
+        test('status: 200', () => {
```

Comment:
This test is covered by the following test. No need to be this granular in my opinion.